src/components/props/uploadProps.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { PropsTableRow, PropsTableRowProps } from '../propsTable'
+
+export const renderFilePreview = (props: PropsTableRowProps) => (
+	<PropsTableRow
+		{...props}
+		prop="renderFilePreview"
+		propType={`(options: RenderFilePreviewOptions<AcceptArtifacts>) => ReactNode`}
+		description="Renders the file preview."
+	/>
+)
+
+export const acceptMimeTypes = (props: PropsTableRowProps) => (
+	<PropsTableRow
+		{...props}
+		prop="acceptMimeTypes"
+		propType={`undefined | string | string[]`}
+		description="Accepted mime types."
+	/>
+)
+
+export const acceptFile = (props: PropsTableRowProps) => (
+    <PropsTableRow
+        {...props}
+        prop="acceptFile"
+        propType={`undefined | ((options: AcceptFileOptions) => boolean | Promise<unknown>)`}
+        description={`
+            A function that is called to check if the file is accepted.
+        `}
+    />
+)
+
+export const urlField = (props: PropsTableRowProps) => (
+    <PropsTableRow
+        {...props}
+        prop="urlField"
+        propType="string"
+        description="The name of the column in Contember schema where to store file url."
```

Comment:
I would love to see link to query language in these fields - maybe even something special in the UI?

Reply:
Link to query language would be nice also in examples. Let's make it great in next version.